release/moloch_update_geo.sh

```diff
@@ -2,25 +2,23 @@
 
 # Variables
 DEST_DIR="${MOLOCH_DIR:-BUILD_MOLOCH_INSTALL_DIR}/etc"
-TIMEOUT=${WGET_TIMEOUT:-30}
+TIMEOUT="${WGET_TIMEOUT:-30}"
 
```

Comment:
Just add a cd $DEST_DIR if you don't want to use $DEST_DIR/... everywhere

Reply:
Resolved :)